### ISSUE TITLE
csshx doesn't see my keys in my local .ssh/  OR csshx doesn't work at all when a valid public key is available

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#11]
```
What steps will reproduce the problem?
1. Set up keyed access to a Linux server.
2. Make sure you can SSH into the the Linux server using the key (not the
password)
3. Try the same command with csshX instead of ssh

What is the expected output? What do you see instead?

I expect to be able to log into the server using the key, not getting
prompted for a password.

Instead, I am prompted for the password, and THEN entering the correct
password does not let me log on (what the...?)


What version of the product are you using? On what operating system?

0.61 on OS X 10.5.7

Please provide any additional information below.

Weird!  I had tried upgrading from 0.59 today.  I reverted back to 0.59 and
everything works fine again.
```

Original issue reported on code.google.com by `nathan.s...@gmail.com` on 3 Jun 2009 at 11:54

**Comment by GoogleCodeExporter** [comment]
```
This is odd - it should work. 

I am using 0.62 here to ssh into Linux machines with keys with no issue. I am 
usually using ssh-agent to do 
this, but I also confirmed this works for me with setting the key via 
--ssh_args.

What arguments are you passing to csshx?

In testing this, I have noticed an issue with the "user@host" syntax. If you 
are using that, does it work with "-l 
user host" instead?
```

Original comment by `gavin.brock` on 4 Jun 2009 at 1:21

**Comment by GoogleCodeExporter** [comment]
```
Ok fixed the username@hostname issue.

If you can, please try the svn version at:

  http://csshx.googlecode.com/svn/trunk/csshX

If you run with the "-d" flag, it will now show you the "ssh" command being 
executed.

If you could copy and paste that into the bug, it might help.

Other thoughts:

     * Do you have a strange shell, or magic environment
     * The path to the ssh key (if specified) must be relative to your home folder. (Tilde ~ in the path will not 
work for now)
```

Original comment by `gavin.brock` on 4 Jun 2009 at 1:40

**Comment by GoogleCodeExporter** [comment]
```
-r57 seems to work!  What I was doing earlier was pretty simple:

csshx user@domain

Technically, I've only actually tested "csshx user@domain:port" now, since I 
don't
have keyed access to anything from home on a standard port, but I verified that 
0.62
_didn't_ work with the port setting, so that's a good sign.  I'll discover for 
sure
whether the non-ported version works as well tomorrow at work.

Thanks for the awesome tool and the quick responses to bugs!
```

Original comment by `nathan.s...@gmail.com` on 4 Jun 2009 at 6:40

**Comment by GoogleCodeExporter** [comment]
```
Oh, and I'm just using the normal bash setup on OS X 10.5.7.  Nothing fancy 
about the
environment, though I have made sure CLICOLOR is set so I get pretty colors 
when I do
"ls" :-)
```

Original comment by `nathan.s...@gmail.com` on 4 Jun 2009 at 6:42

**Comment by GoogleCodeExporter** [comment]
```
Any joy with "-d" debug?  I still haven't been able to reproduce this here.
```

Original comment by `gavin.brock` on 10 Jun 2009 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
-r57 released in csshX v0.63. Let me know if you have any further issues.
```

Original comment by `gavin.brock` on 20 Jun 2009 at 4:34
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
0.63 works great.  Thanks. :-)
```

Original comment by `nathan.s...@gmail.com` on 23 Jun 2009 at 9:42
